tests/documentation/test_files/DocumentationStyleBear/bad_file4.py.test.correct

```diff
@@ -0,0 +1,12 @@
+def myfunc(a, b, x):
+    """
+
```

Comment:
Text must begin in the 1st line (technically 2nd line), but it can be empty.

I would file an issue and add new functionality after merge